api-server/src/main/java/org/opennms/devjam2022/apiserver/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package org.opennms.devjam2022.apiserver.controller;
+
+import java.util.List;
+import org.opennms.devjam2022.apiserver.impl.IUserService;
+import org.opennms.devjam2022.apiserver.model.UserRole;
+import org.opennms.devjam2022.apiserver.model.UserWithRoles;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/user")
+public class UserController {
+
+    @Qualifier("InMemoryUserService")
+    @Autowired
+    IUserService userService;
+
+    @GetMapping("/users")
+    public List<UserWithRoles> all() {
```

Comment:
When we use Sprint webflux, it is sprint reactive and the return type will be Flux<User> for the list of users, Monon<User> for the single suer.

Reply:
Good catch. Thanks. Refactored. Issue #11 created for the future reference